### DISCUSSION TITLE
How do I use it inside the readme.md?

### DISCUSSION THREAD

**Discussion by ArshErgon** [remarkjs/remark#1050] [Q&A] [answered]
I quite new to remark, I am come here because of some issue needs to be fixed (https://app.codacy.com/gh/jmakhack/myanimelist-cli/issues?categoryType=CodeStyle).
```
problem : _If you've any questions, feel free to reach out to <jmakhack@protonmail.com>._

Warn for autolinks without protocol. Autolinks are URLs enclosed in < (less than) and > (greater than) characters.
Fix
remark-stringify adds a protocol where needed. 
```
how do I use remark-stringify on this? by creating a new file and writing the script which is mentioned in the docs? but what if I have multiple errors which uses stringify or lists?

**Comment by JounQin** [comment, marked as answer]
It seems you're using an online service which is using https://github.com/remarkjs/remark-lint inside.

**Reply by ArshErgon** [reply]
Well, It's an issue which I need to solve the repo owner didn't respond, and it's the first time I heard about remark.

**Reply by JounQin** [reply]
You should ask the repo owner to disable code patterns from `remark-lint` tool on codacy

<img width="1763" alt="image" src="https://user-images.githubusercontent.com/8336744/192137308-6035620d-36e2-4454-8dff-f3d91b23e16e.png">

Or you should follow the code patterns used by codacy by default if you/the repo owner still want to use it on codacy.

**Reply by ArshErgon** [reply]
thanks
